Assets/MixedRealityToolkit/_Core/Utilities/Scripts/DebugUtilities.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.InputSystem.Focus;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    public static class DebugUtilities
```

Comment:
Not so sure about this one

Reply:
Also, I plan on updating any LogFormat's to use string interpolation.

Reply:
Fixed, LogFormat and ErrorFormat begone everywhere now 🗡

Reply:
Could we split the debug stuff into a separate PR for later?

Reply:
That would be terribly annoying, but sure.
I'll leave the Debug class there, just revert the implementations.  But still unsure as to your objections, since it's not removing the calls, just wrapping them within our own implementation.

Reply:
I think including it in it's own pr so we can get more feedback from the community.

The initial input PR might already be a lot to cover.  I only included utility classes that were needed to satasfy the input systems dependencies.